frontend/packages/dev-console/src/components/topology/shapes/ResourceIcon.scss

```diff
@@ -9,6 +9,6 @@
 .odc-resource-icon-deployment,
 .odc-resource-icon-daemonset,
 .odc-resource-icon-deploymentconfig {
-  cursor: pointer;
+  cursor: auto;
```

Comment:
No need to add cursor property. Default is fine.

Reply:
With Default, we won't get the hover effect as we have for other resources badges. So added.

Reply:
Got it. Thanks.